server/src/main/java/no/nav/foreldrepenger/vtp/server/rest/auth/UserRepository.java

```diff
@@ -0,0 +1,40 @@
+package no.nav.foreldrepenger.vtp.server.rest.auth;
+
+import javax.naming.NamingEnumeration;
+import javax.naming.NamingException;
+import javax.naming.directory.SearchControls;
+import javax.naming.directory.SearchResult;
+import javax.naming.ldap.InitialLdapContext;
+import javax.naming.ldap.LdapName;
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+
+public class UserRepository {
+    public static List<SearchResult> getAllUsers() throws NamingException {
+        Hashtable<String, String> props = new Hashtable<>();
```

Comment:
Har du strengt tatt behov for å bruke HashTable? De fleste er mer vant til å se HashMap i stedet.

`        HashMap<String, String> props = new HashMap<>();`